src/test/java/com/google/api/codegen/testdata/library.proto

```diff
@@ -105,7 +105,7 @@ service LibraryService {
 
   // Adds comments to a book
   rpc AddComments(AddCommentsRequest) returns (google.protobuf.Empty) {
-    option (google.api.http) = { post: "/v1/{name=shelves/*}/comments" body: "*" };
+    option (google.api.http) = { post: "/v1/{name=bookShelves/*}/comments" body: "*" };
```

Comment:
Hmmm yes you are right but since I already made the change..I will leave it as is for now.